### ISSUE TITLE
Review article on “WCAG 3.0 proposes new contrast method”

### ISSUE THREAD

**Issue by mrosvik** [digdir/designsystemet#3997]
We received feedback that the article “WCAG 3.0 proposes new contrast method” may be somewhat outdated and lacks source references.

It was also pointed out that it has not been decided whether WCAG 3.0 will actually use APCA, so our description should be reviewed and possibly adjusted.

Suggested actions:
- Review the article and check the content against the latest WCAG 3.0 status
- Add relevant source references
- Adjust how APCA is described

---

Big thanks to Anders Ekkje Slettebø for pointing this out 🙌

**Comment by Camulos** [comment]
Endringer foreslått for norsk versjon i pr [3999](https://github.com/digdir/designsystemet/pull/3999)
